source/_components/bmw_connected_drive.markdown

```diff
@@ -50,8 +50,8 @@ bmw_connected_drive:
       description: "The region of your Connected Drive account. Please use of these values: `north_america`, `china`, `rest_of_world`"
       required: true
       type: string
-    services:
-      description: Enables or disables the services of the vehicles including the lock. Default true (Services enabled)
+    read_only:
+      description: Enables or disables the services of the vehicles including the lock. Default false (Services enabled)
```

Comment:
Please use the `default:` key to clarify what the default is -> [documentation](https://developers.home-assistant.io/docs/en/documentation_create_page.html).